src/main/java/internet/shop/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,57 @@
+package internet.shop.dao.impl;
+
+import internet.shop.dao.ManufacturerDao;
+import internet.shop.db.Storage;
+import internet.shop.lib.Dao;
+import internet.shop.model.Manufacturer;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        Storage.addProduct(manufacturer);
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        List<Manufacturer> dbManufacturer = Storage.getDbManufacturer();
+        for (int i = 0; i < dbManufacturer.size(); i++) {
+            if (id == dbManufacturer.get(i).getId()) {
```

Comment:
what is the correct way to compare objects? 
(let's use stream instead, code will get shorter)

Reply:
done